### DISCUSSION TITLE
Volunteer participation in exoplanet searches: microlensing

### DISCUSSION THREAD

**Discussion by eddiemorris135** [lightkurve/lightkurve#1274] [General]
Hi All,

I would like to contribute as a volunteer for finding exoplanets via microlensing via some python work. Although I am only a python novice and currently going through the https://docs.lightkurve.org/ tutorials, I would love to try. 

I am aware of zooniverse but I would like more interaction with python to enhance my python skills in parallel to exoplanet studies. 
Are there some volunteer sources I could try?

Thanks!

**Comment by orionlee** [comment]
https://www.planethuners.coffee/ has a series of videos about exoplanets and using Python / lightkurve to analyze lightcurve data from TESS. 

While it focuses on transit method, many topics involved are applicable for analyzing lightcurve data  in general.
